src/Helpers/JWKFetcher.php

```diff
@@ -17,11 +17,11 @@
 class JWKFetcher
 {
     /**
-     * How long should the cache persist? Set to 10 minutes.
+     * How long should the cache persist? Defaults to 10 minutes.
      *
      * @see https://www.php-fig.org/psr/psr-16/#12-definitions
      */
-    const CACHE_TTL = 600;
+    private $ttl = 600;
```

Comment:
my understanding is that PHP class constants are public by default. If that's the case, then this would be a breaking change, right?

Reply:
You’re right about that const removal being a breaking change, unfortunately. Since users have been writing custom cache logic to do workarounds until we added the cache controls this PR introduces, it's possible some may have relied on that const one way or another.

Considering this will ship alongside some other significant changes being queued up (although none to this point have been breaking) wrapping it up as a major release feels alright to me, unless you have any thoughts on handling this better?

Reply:
Per our Slack convo — I've reverted this change; CACHE_TTL now reflects the default TTL that JWKFetcher will use, assigned to $ttl when the class is constructed.